api/models/weather.py

```diff
@@ -116,3 +117,7 @@ class Weather(Model):
 
     class Options:
         serialize_when_none = False
+
+    @serializable(type=StringType, serialized_name='id')
+    def id(self):
+        return unidecode(f'{self.city.name.upper()}')
```

Comment:
At PassFort's team we use [UUID](https://docs.python.org/3/library/uuid.html). Please, take a look.